myuw/data/resource_link_import.csv

```diff
@@ -1,4 +1,4 @@
-Category,Subcategory,affiliation,Central (not campus specific),Central Title,Seattle,Seattle (link text),Bothell (destination URL),Bothell (link text),Tacoma,Tacoma Title,Open in a new tab?
+Category,Subcategory,Working Affiliation,Central (not campus specific),Central Title,Seattle (destination URL),Seattle (link text),Bothell (destination URL),Bothell (link text),Tacoma,Tacoma Title,Open in new tab?
```

Comment:
@DannyNineNineNine Not sure if this will break anything, but I don't think you need to edit the first line of the CSV to make these changes. I would probably just change these back to what they were.